canonical-web-and-design/ubuntu.com
# Use logic to hide/show latest release

## Done

- Use logic to hide/show latest release

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/alternative-downloads and http://0.0.0.0:8001/download/alternative-downloads#download
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that logic hides or shows the latest if you change the releases.yaml
